### PR TITLE
icons etc.

### PR DESCRIPTION
**Pull request by hlyford11**
- small icons for trending quizzes on index page
- player card partials by each league

### CODE REVIEW

File: public/stylesheets/style.css [hunk 1]
```diff
@@ -38,6 +38,7 @@ a {
 .rb-white {
   color: #ffffff; 
 }
+/*
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: why not just delete??

File: public/stylesheets/style.css [hunk 2]
```diff
@@ -58,14 +59,66 @@ a {
   background-color: #fff;
   vertical-align: -7px;
 }
-.badge-roster {
-
+*/
+.badge-small- {
+  display: inline-block;
+  min-width: 10px;
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: u dont need min-width since u are setting a width below anyway

File: public/stylesheets/style.css [hunk 3]
```diff
@@ -58,14 +59,66 @@ a {
...
+  vertical-align: baseline;
+  border-radius: 50%;
+  display: inline-block;
+  width: 25px;
+  height: 25px;
+  margin: 0 2px 0 5px;    
+  content: url(https://cdn4.iconfinder.com/data/icons/ios7-essence/22/editor_clipboard_checkmark_done-512.png) no-repeat 0 0;
+  background-size: 100%;
+  background-color: #fff;
+  vertical-align: -7px;
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: you already did vertical align above smh

File: public/stylesheets/style.css [hunk 4]
```diff
@@ -58,14 +59,66 @@ a {
...
 }
-.badge-leaders {
-
+.badge-small-leaders {
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: this is sharing the exact styles as the one above. just make one class

File: public/stylesheets/style.css [hunk 5]
```diff
@@ -58,14 +59,66 @@ a {
...
+  height: 25px;
+  margin: 0 2px 0 5px;    
+  content: url(http://cdn.mysitemyway.com/etc-mysitemyway/icons/legacy-previews/icons/magic-marker-icons-culture/115276-magic-marker-icon-culture-crown-five-points2.png) no-repeat 0 0;
+  background-size: 100%;
+  background-color: #fff;
+  vertical-align: -7px;
 }
-.badge-goats {
-
+.badge-small-goats {
```
Comment (blakeparkinson) [inline public/stylesheets/style.css]: exact styles...one class

File: routes/leaguesearch.js [hunk 6]
```diff
@@ -154,6 +154,25 @@ var randImg = function() {
   image = path + image;
   return image;    
 }
+*/
+
+var imageByLeague = function(league) {
+  if (league === 'nba') {
```
Comment (blakeparkinson) [inline routes/leaguesearch.js]: this doesnt look nice. Can you use a switch statement to make it more readable

File: views/partials/mlb_player_card.hbs [hunk 7]
```diff
@@ -0,0 +1,37 @@
+<div>Bats: <span>Throws: </span> </div>
+<table id="stats-table" class="player-card-table">
+  <tr>
+    <th style="background-color: {{primary_hex}}">AVG</th>
```
Comment (blakeparkinson) [inline views/partials/mlb_player_card.hbs]: are u sure, you can set primary_hex on the tr, so you dont have to do it on each th?

File: views/partials/nfl_player_card.hbs [hunk 8]
```diff
@@ -0,0 +1,37 @@
+<table id="stats-table" class="player-card-table">
+  <tr>
+    <th style="background-color: {{primary_hex}}">PPG</th>
```
Comment (blakeparkinson) [inline views/partials/nfl_player_card.hbs]: same comment as above

File: views/partials/soccer_player_card.hbs [hunk 9]
```diff
@@ -0,0 +1,37 @@
+<table id="stats-table" class="player-card-table">
+  <tr>
+    <th style="background-color: {{primary_hex}}">PPG</th>
```
Comment (blakeparkinson) [inline views/partials/soccer_player_card.hbs]: same comment i had earlier

File: public/stylesheets/style.css (hunk 4 above)
Comment (hlyford11) [inline public/stylesheets/style.css]: these may change to match image so im going to leave for now. i.e. the same styles might not work for the different images

File: views/partials/mlb_player_card.hbs (hunk 7 above)
Comment (hlyford11) [inline views/partials/mlb_player_card.hbs]: this is just temporary. not even getting hex for baseball yet. just copied the card over.

File: routes/leaguesearch.js (hunk 6 above)
Comment (hlyford11) [inline routes/leaguesearch.js]: done.

**Comment by blakeparkinson** [comment]
ok

**Pull request merged**